web/backend/src/Server.ts

```diff
@@ -200,43 +156,22 @@ app.post('/api/logout', (req, res) => {
   });
 });
 
-// This function helps us convert the double list of the authorization
-// returned by the casbin function getFilteredPolicy to a map that link
-// an object to the action authorized
-// list[0] contains the policies so list[i][0] is the sciper
-// list[i][1] is the subject and list[i][2] is the action
-function setMapAuthorization(list: string[][]): Map<String, Array<String>> {
-  const userRights = new Map<String, Array<String>>();
-  for (let i = 0; i < list.length; i += 1) {
-    const subject = list[i][1];
-    const action = list[i][2];
-    if (userRights.has(subject)) {
-      userRights.get(subject)?.push(action);
-    } else {
-      userRights.set(subject, [action]);
-    }
-  }
-  console.log(userRights);
-  return userRights;
-}
-
 // As the user is logged on the app via this express but must also
 // be logged into react. This endpoint serves to send to the client (actually to react)
 // the information of the current user.
-app.get('/api/personal_info', (req, res) => {
-  authEnforcer.getFilteredPolicy(0, String(req.session.userId)).then((AuthRights) => {
-    res.set('Access-Control-Allow-Origin', '*');
-    if (req.session.userId) {
-      res.json({
-        sciper: req.session.userId,
-        lastName: req.session.lastName,
-        firstName: req.session.firstName,
-        isLoggedIn: true,
-        authorization: Object.fromEntries(setMapAuthorization(AuthRights)),
-      });
-    } else {
-      res.status(401).send();
-    }
+app.get('/api/personal_info', async (req, res) => {
+  if (!req.session.userId) {
+    res.status(401).send('UnAuthenticated');
```

Comment:
please change 'UnAuthenticated' to 'Unauthenticated'

Reply:
apart from that looks great :+1:

Reply:
done